app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -86,6 +100,54 @@ public Camera2() {
 
 	}
 
+	private boolean inRange(int SubjectValue, int High, int Low) {
+		// In case of a 360
+		SubjectValue =  (360 + (SubjectValue % 360)) % 360;
+		if (High < Low) {
+			return High <= SubjectValue && SubjectValue <= Low;
+		} else {
+			return High <= SubjectValue || SubjectValue <= Low;
+		}
+	}
+
+	private void setFlashIcon() {
+		switch (state) {
+			case 0:
+				flash.setImageResource(R.drawable.flash_on);
```

Comment:
Ahh right! The case 0 will be handled by default case right? :D I'll fix it! :+1: